mypy/visitor.py

```diff
@@ -1,10 +1,10 @@
 """Generic abstract syntax tree node visitor"""
 
 from abc import abstractmethod
-from typing import TypeVar, Generic
+from typing import TypeVar, Generic, TYPE_CHECKING
```

Comment:
Well, this should be imported from `typing_extensions`, see imports in other modules (because of Python 3.5.1).

Reply:
Argh, Python 3.5 strikes yet again...

I really wish we could drop support for it early. It sometimes feels like I get bitten by some Python 3.5 related issue in every other PR I try submitting...